daprdocs/content/en/getting-started/quickstarts/configuration-quickstart.md

```diff
@@ -0,0 +1,634 @@
+---
+type: docs
+title: "Quickstart: Configuration"
+linkTitle: Configuration
+weight: 76
+description: Get started with Dapr's Configuration building block
+---
+
+LEt's take a look at Dapr's [Configuration building block]({{ ref configuration-api-overview }}). A configuration item is often dynamic in nature and tightly coupled to the needs of the application that consumes it. Configuration items are key/value pairs containing configuration data, such as:
+- App ids
+- Partition keys
+- Database names, etc
+
+In this quickstart, you'll run an `order-processor` microservice that utilizes the Configuration API. The service:
+1. Gets configuration items from the configuration store.
+1. Subscribes for configuration updates.
+
+<!-- 
+Include a diagram or image, if possible. 
+-->
+
+Select your preferred language-specific Dapr SDK before proceeding with the Quickstart.
+
+{{< tabs "Python" "JavaScript" ".NET" "Java" "Go" >}}
+ <!-- Python -->
+{{% codetab %}}
+
+### Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+<!-- IGNORE_LINKS --> 
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+<!-- END_IGNORE -->
+
+### Step 1: Set up the environment
+
+Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/configuration).
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+Once cloned, open a new terminal and run the following command to set values for configuration items `orderId1` and `orderId2`.
+
+```bash
+docker exec dapr_redis redis-cli MSET orderId1 "101" orderId2 "102"
+```
+
+### Step 2: Run the `order-processor` service
+
+From the root of the Quickstarts clone directory, navigate to the `order-processor` directory.
+
+```bash
+cd ./configuration/python/sdk/order-processor
+```
+
+Install the dependencies:
+
+```bash
+pip3 install -r requirements.txt
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --resources-path ../../../components/ --app-port 6001 -- python3 app.py
+```
+
+> **Note**: Since Python3.exe is not defined in Windows, you may need to use `python app.py` instead of `python3 app.py`.
+
+The expected output:
+
+```
+
+```
+
+### Step 3: Update configuration item values
+
+Once the app has unsubscribed, try updating the configuration item values. Change the `orderId1` and `orderId2` values using the following command:
+
+```bash
+docker exec dapr_redis redis-cli MSET orderId1 "103" orderId2 "104"
+```
+
+Run the `order-processor` service again:
+
+```bash
+dapr run --app-id order-processor --resources-path ../../../components/ --app-port 6001 -- python3 app.py
+```
```

Comment:
The idea was to keep this step of updating values as optional and if the user updates the value while the app is still running, the app will be able to observe the changes. After the timeout, the app will unsubscribe and exit.

Reply:
@sarsharma Ah ok - I'll make it an optional step then